.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
+  "extends": "airbnb-base",
```

Comment:
I think ESLint also need to extend Prettier to avoid conflict.
I'm not quite sure about tho, so guys please come in here.

Reply:
I will read about that, thank you.

Reply:
Oh sorry for not being clear, I meant Linting conflict. Not merge conflict.